test/integration/smoke/test_deploy_vm_root_resize.py

```diff
@@ -212,35 +212,29 @@ def test_00_deploy_vm_root_resize(self):
         # 3. Rejects non-supported hypervisor types
```

Comment:
I will try to add some tests for these, in test/integration/component/test_resource_limits.py didn't see any PS resource limit tests.